SignalMessaging/categories/UIDevice+featureSupport.swift

```diff
@@ -34,6 +34,11 @@ public extension UIDevice {
         }
     }
 
+    @objc
+    public var isShorterThanIPhone5: Bool {
+        return UIScreen.main.nativeBounds.height < 1136
```

Comment:
Currently there's no device for which this would produce a different result, but I think conceptually you're right, it does make more sense to measure points.

Reply:
I merged this change in b0978abd0163ff0534863a9fb50892beb8177f9c